src/OrbitGl/CaptureWindow.cpp

```diff
@@ -113,8 +113,21 @@ void CaptureWindow::PreRender() {
   }
 
```

Comment:
I'd add introspection scopes here so that we can see how long CaptureWindow::PreRender takes but also how many times the loop executes.

Reply:
I've created a bug for this - same as below, I'd like to fix the weird "Finalizing capture" condition before.